ocode_python/tools/bash_tool.py

```
@@ -781,67 +768,75 @@ async def execute(self, **kwargs: Any) -> ToolResult:
-                    # Add shebang if needed (Unix only)
-                    if platform.system() != "Windows":
-                        if script_type == "bash":
-                            script_file.write("#!/bin/bash\nset -e\n\n")
-                        elif script_type == "python":
-                            script_file.write("#!/usr/bin/env python3\n\n")
-                        elif script_type in ["node", "javascript"]:
-                            script_file.write("#!/usr/bin/env node\n\n")
-
-                    script_file.write(script_content)
+                # Add shebang if needed (Unix only)
+                import platform
```

Pylint (3.3.7)

[convention] Import outside toplevel (platform)

(C0415)